src/material/form-field/form-field.html

```diff
@@ -96,25 +96,28 @@
 </div>
 
 <div
-  class="mat-mdc-form-field-subscript-wrapper mat-mdc-form-field-bottom-align"
-  [class.mat-mdc-form-field-subscript-dynamic-size]="subscriptSizing === 'dynamic'"
+    class="mat-mdc-form-field-subscript-wrapper mat-mdc-form-field-bottom-align"
+    [class.mat-mdc-form-field-subscript-dynamic-size]="subscriptSizing === 'dynamic'"
 >
-  @switch (_getDisplayedMessages()) {
-    @case ('error') {
-      <div class="mat-mdc-form-field-error-wrapper">
+  @let subscriptMessageType = _getSubscriptMessageType();
+
+  <div aria-atomic="true" aria-live="polite" 
```

Comment:
Should we leave a comment here saying why the `aria-live` is on the container, rather than the individual message?